sigmadb.py

```diff
@@ -91,11 +91,18 @@ def main():
 
                 action = ""
                 while action != "1":
-                    action = input_utils.get_input_matching("1 - exit\n2 - browse movies\n> ", regex="[12]")
+                    action = input_utils.get_input_matching("1 - exit\n2 - browse movies\n4 - create collection\n5 - modify collection\n6 - Display collection ", regex='[12456]')
```

Comment:
This is probably going to cause merge conflicts. Can you pull master into your branch to manually merge the two? Lmk if you're not sure how to do that.
The command should be: `git pull master`